docs/import/create_score_asset.js

```diff
@@ -3,67 +3,107 @@ import {getBackupScoreAssetPath, getScoreAssetPath} from '../resources.js';
 import {computeAndSaveBounds} from './center.js';
 import {cdcGeoidKey, censusBlockGroupKey, censusGeoidKey, tigerGeoidKey} from './import_data_keys.js';
 
-export {createScoreAsset, setStatus};
-
+export {
+  createScoreAssetForFlexibleDisaster,
+  createScoreAssetForStateBasedDisaster,
+  setStatus,
+};
 // For testing.
 export {backUpAssetAndStartTask};
 
 /**
- * For the given feature representing a block group, add properties for
- * total # buildings in the block group and # damaged buildings in the block
- * group.
+ * Given a dictionary of building counts per district, attach the count to each
+ * district in the collection.
+ * @param {ee.FeatureCollection} featureCollection The features'
+ *     {@link geoidTag} are the keys of `buildingsHisto`
+ * @param {ee.Dictionary} buildingsHisto Each entry gives how many buildings are
+ *     in the feature whose {@link geoidTag} property is the entry's key
+ * @return {ee.FeatureCollection}
+ */
+function combineWithBuildings(featureCollection, buildingsHisto) {
+  return featureCollection.map((f) => {
+    const geoId = f.get(geoidTag);
+    return f.set(
+        buildingCountTag,
+        ee.Algorithms.If(
+            buildingsHisto.contains(geoId), buildingsHisto.get(geoId),
+            ee.Number(0)));
+  });
+}
+
+/**
+ * Given an in-progress score asset, adds in the damage percentage. The damage
+ * asset may include undamaged buildings, in which case we must filter them out
+ * via the `damageLevelsKey` and `noDamageValue` parameters.
+ * @param {ee.FeatureCollection} featureCollection Score asset being created
+ * @param {ee.FeatureCollection} damage Damage collection, which may includes
+ *     undamaged buildings
+ * @param {?string} damageLevelsKey Property in damage collection whose value
+ *     can identify undamaged buildings
+ * @param {?string} noDamageValue Value for `damageLevelsKey` that indicates a
```

Comment:
Something we definitely need to document - we expect only one damage/undamaged col with all possible states

Reply:
I think we allow a slightly more permissive structure, although we have to be careful how we document it: if there is a column per damage type, then the user can set the damageLevelsKey to the column for "undamaged", and set noDamageValue to "true" or whatever the sentinel for that column is. So I think this should allow for all our currently known types of damage assets.

Reply:
Oh great point. Yeah this is going to be a doozy to document. Maybe the best way is to provide a couple different supported forms.